root/edit/EditData.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import EditLink from '../static/scripts/common/components/EditLink';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../static/scripts/common/i18n/commaOnlyList';
+import {bracketedText} from '../static/scripts/common/utility/bracketed';
+import {formatPluralEntityTypeName}
+  from '../static/scripts/common/utility/formatEntityTypeName';
+import {getEditStatusName} from '../utility/edit';
+
+type Props = {
+  +edit: {...EditT, +id: number},
+  +relatedEntities: {[type: CoreEntityTypeT]: {
+    [entityId: string]: CoreEntityT,
+  }},
+};
+
+const EditIndex = ({
+  edit,
+  relatedEntities,
+}: Props): React.Element<typeof Layout> => {
+  const title = texp.l('Edit data for edit #{id}', {id: edit.id});
+  const jsonData = JSON.parse(edit.raw_data);
+  const displayData = jsonData
+    ? JSON.stringify(jsonData, null, 2)
+    : edit.raw_data;
+
+  return (
+    <Layout fullWidth title={title}>
+      <h1>{title}</h1>
+
+      <p>
+        <strong>{('Type:')}</strong>
+        {' '}
+        {l(edit.edit_name)}
+        {' '}
+        {bracketedText(edit.edit_type)}
+      </p>
+
+      <p>
+        <strong>{('Status:')}</strong>
+        {' '}
+        {getEditStatusName(edit)}
+        {' '}
+        {bracketedText(edit.status)}
+      </p>
+
+      <p><strong>{l('Data:')}</strong></p>
+      <pre id="edit-raw-data">{displayData}</pre>
+
+      {Object.keys(relatedEntities).length ? (
+        <>
+          <p><strong>{l('Related entities:')}</strong></p>
+          <ul>
+            {Object.keys(relatedEntities).sort().map((type, index) => {
```

Comment:
Since we have to use the keys twice and sort them, it seems worth putting them in a variable first:
```js
const relatedEntityTypes = Object.keys(relatedEntities).sort();
```

```suggestion
      {relatedEntityTypes.length ? (
        <>
          <p><strong>{l('Related entities:')}</strong></p>
          <ul>
            {relatedEntityTypes.map((type, index) => {
```